### PR TITLE
[webnfc] Add NDEFRecord and NDEFMessage constructors in idl tests

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@leonhsl, PTAL.

File: web-nfc/idlharness.https.window.js [hunk 1]
```diff
@@ -19,6 +23,8 @@ idl_test(
     idl_array.add_objects({
       NFCWriter: ['new NFCWriter();'],
       NFCReader: ['new NFCReader();'],
+      NDEFRecord: [`new NDEFRecord(${JSON.stringify(record)});`],
+      NDEFMessage: [`new NDEFMessage([${JSON.stringify(record)}]);`],
```
Comment (leonhsl) [inline web-nfc/idlharness.https.window.js]: [${JSON.stringify(record)}] --> ${JSON.stringify(message)} ?

**Review by leonhsl — CHANGES_REQUESTED** [review]

File: web-nfc/idlharness.https.window.js (hunk 1 above)
Comment (Honry) [reply web-nfc/idlharness.https.window.js]: Ops, my bad. Fixed.

File: web-nfc/idlharness.https.window.js [hunk 2]
```diff
@@ -19,6 +23,8 @@ idl_test(
     idl_array.add_objects({
       NFCWriter: ['new NFCWriter();'],
       NFCReader: ['new NFCReader();'],
+      NDEFRecord: [`new NDEFRecord(${JSON.stringify(record)});`],
+      NDEFMessage: [`new NDEFMessage([${JSON.stringify(message)}]);`],
```
Comment (leonhsl) [inline web-nfc/idlharness.https.window.js]: No [] for embracing ${JSON.stringify(message)}

**Review by leonhsl — CHANGES_REQUESTED** [review]

File: web-nfc/idlharness.https.window.js (hunk 2 above)
Comment (Honry) [reply web-nfc/idlharness.https.window.js]: Done. 😳

**Review by leonhsl — APPROVED** [review]

**Comment by Honry** [comment]
@leonhsl, thanks!

**Pull request merged**